src/main/java/com/github/vkorobkov/jfixtures/JFixturesOld.java

```diff
@@ -8,8 +8,9 @@
 import com.github.vkorobkov.jfixtures.sql.dialects.MySql;
 import com.github.vkorobkov.jfixtures.sql.dialects.Sql99;
 
-public final class JFixtures {
-    private JFixtures() {
+@Deprecated
```

Comment:
Hm, I guess we need to add some comments above the annotation like:

```
/**
 * Does some thing in old style.
 *
 * @deprecated use {@link #new()} instead. This class will be changed in release 1.0.36..
 */
```
Why do we need to rename this class? So when I use 1.0.35 release https://github.com/rodionovsasha/CommentService it will break the build..

What do you think?

Reply:
Not sure, release 1.0.35 is going to have breaking changes - `JFixturesOld` will be just deleted. But I can add a comment for that time that we will develop 1.0.35 ❤️